### PR TITLE
FLU: Convert Khepera to FLU

### PR DESCRIPTION
**Pull request by lukicdarkoo**
- [x] (738) robots/k-team/khepera4/protos/Khepera4InfraredSensor.proto
- [x] (739) robots/k-team/khepera4/protos/Khepera4.proto
- [x] (740) robots/k-team/khepera1/protos/Khepera1.proto
- [x] (741) robots/k-team/khepera3/protos/Khepera3_USSensor.proto
- [x] (742) robots/k-team/khepera3/protos/Khepera3_DistanceSensor.proto
- [x] (743) robots/k-team/khepera3/protos/Khepera3_GroundSensor.proto
- [x] (744) robots/k-team/khepera3/protos/Khepera3.proto
- [x] (745) robots/k-team/khepera3/protos/Khepera3_Gripper.proto
- [x] (747) robots/k-team/khepera2/protos/Khepera2.proto
- [x] (286) projects/robots/k-team/khepera4/worlds/khepera4.wbt
- [x] (287) projects/robots/k-team/khepera1/worlds/khepera_k213.wbt
- [x] (288) projects/robots/k-team/khepera1/worlds/khepera_gripper.wbt
- [x] (289) projects/robots/k-team/khepera1/worlds/khepera_pipe.wbt
- [x] (290) projects/robots/k-team/khepera1/worlds/khepera_tcpip.wbt
- [x] (291) projects/robots/k-team/khepera1/worlds/khepera_gripper_camera.wbt
- [x] (292) projects/robots/k-team/khepera1/worlds/khepera_kinematic.wbt
- [x] (293) projects/robots/k-team/khepera1/worlds/khepera.wbt
- [x] (294) projects/robots/k-team/khepera3/worlds/khepera3_gripper.wbt
- [x] (295) projects/robots/k-team/khepera3/worlds/khepera3.wbt
- [x] (297) projects/robots/k-team/khepera2/worlds/khepera2.wbt

### CODE REVIEW

File: projects/robots/k-team/khepera1/worlds/khepera_k213.wbt [hunk 1]
```diff
@@ -292,6 +295,8 @@ Khepera1 {
           ]
         }
         Transform {
+          translation 0 -0.014 0.01
+          rotation 0 1 0 0
```
Comment (BenjaminDeleze) [inline projects/robots/k-team/khepera1/worlds/khepera_k213.wbt:299]:
```suggestion
```

File: projects/robots/k-team/khepera1/worlds/khepera_k213.wbt [hunk 2]
```diff
@@ -266,17 +268,18 @@ DEF PANEL Solid {
...
   controller "khepera_k213"
   extensionSlot [
     DEF K213 Camera {
-      translation 0 0.053 0
+      translation 0 0.013 0.04
       rotation 1 0 0 0
       children [
         DEF K213_GREEN_CYLINDER Transform {
           translation 0 -0.013 0
+          rotation 0 1 0 0
```
Comment (BenjaminDeleze) [inline projects/robots/k-team/khepera1/worlds/khepera_k213.wbt:282]:
```suggestion
```

File: projects/robots/k-team/khepera3/worlds/khepera3_gripper.wbt [hunk 3]
```diff
@@ -23,47 +22,49 @@ WorldInfo {
...
+  orientation 0.20022020315551092 0.3500108693031503 -0.9150979519253767 1.1173475389874992
+  position -0.9048449902243705 0.9353685043387764 0.5474274271199836
   follow "Khepera III"
 }
 TexturedBackground {
 }
 TexturedBackgroundLight {
 }
 RectangleArena {
+  rotation 0 1 0 0
```
Comment (BenjaminDeleze) [inline projects/robots/k-team/khepera3/worlds/khepera3_gripper.wbt:34]:
```suggestion
```

**Review by BenjaminDeleze — CHANGES_REQUESTED** [review]
- Some sensors of the Khepera 3 are using cylinder and needs to be converted
- `Khepera.wbt`: the robot is oriented the opposite way. Is this done on purpose?

**Comment by lukicdarkoo** [comment]
> Some sensors of the Khepera 3 are using cylinder and needs to be converted

Fixed

> * `Khepera.wbt`: the robot is oriented the opposite way. Is this done on purpose?

No special reason, it is just a nice orientation `0 0 1 0` (default)

**Review by BenjaminDeleze — APPROVED** [review]
Thank you!

**Pull request merged**